demisto_sdk/commands/common/hook_validations/docker.py

```diff
@@ -410,3 +419,29 @@ def _get_latest_commit(commits_url, docker_image_name):
                             'Please use only images that are already in Iron Bank, or upload your image to it.')
 
         return last_successful_pipelines[0]['sha']
+
+    @staticmethod
+    def get_deprecated_dockers_list():
+        """
+        Get the deprecated docker images from dockerfiles repo
+        returns: Dict
+        """
+        try:
+            dockers_request = requests.get(DEPRECATED_DOCKER_IMAGE_LIST, verify=False)
+            dockers_request.raise_for_status()
+            deprecated_dockers_json = dockers_request.json()
+            return deprecated_dockers_json
+        except Exception as exc:
+            exc_msg = str(exc)
+            if isinstance(exc, requests.exceptions.ConnectionError):
+                exc_msg = f'{exc_msg[exc_msg.find(">") + 3:-3]}.\n' \
+                          f'This may happen if you are not connected to the internet.'
+            raise Exception(f'Could not get latest demisto-sdk version.\nEncountered error: {exc_msg}')
+
+    def is_docker_image_deprecated(self, docker_image_name):
```

Comment:
Let's also add docstting
```suggestion
    def is_docker_image_deprecated(self, docker_image_name) -> Tuple[str, str]:
```